test/methods.js

```diff
@@ -230,7 +230,7 @@ describe('nemo-view @methods@', function () {
     util.waitForJSReady(nemo).then(function () {
       start = Date.now();
     });
-    nemo.view._waitVisible('#foo.bar.brrao', 3000).then(function (find) {
+    nemo.view._waitVisible('#foo.bar.brrao', 3000, 'Element did not load for specified timeout').then(function (find) {
```

Comment:
Instead of these changes to the tests, can you create a new test (or tests) to specifically test that the custom message is thrown in the error?